src-executables/Main.hs

```diff
@@ -6,16 +6,38 @@ import           Poseidon.CmdFStats    (FStatSpec (..), FstatsOptions (..),
 import           Poseidon.CmdList      (ListEntity (..), ListOptions (..),
                                         runList)
 import           Poseidon.CmdSummarise (SummariseOptions(..), runSummarise)
-
+import           Poseidon.CmdValidate  (ValidateOptions(..), runValidate)
 import           Data.ByteString.Char8 (pack, splitWith)
-import           Options.Applicative   as OP
+import Options.Applicative as OP
+    ( Alternative((<|>), many, some),
+      briefDesc,
+      command,
+      eitherReader,
+      flag',
+      help,
+      info,
+      long,
+      metavar,
+      option,
+      progDesc,
+      short,
+      str,
+      strOption,
+      subparser,
+      switch,
+      value,
+      execParser,
+      helper,
+      Parser,
+      ParserInfo )
```

Comment:
Warum importierst Du denn die ganzen Sachen einzeln? Ich denke in so einem Fall sollte man eher `import qualified XX as YY` nehmen. Kann es sein dass wir einfach das `qualified` oben vergessen hatten?

Reply:
Ich hab das grad mal entfernt.